### PR TITLE
Version Reporting

### PR DESCRIPTION
**Pull request by vasukulkarni**
Fixes: https://github.com/red-hat-storage/ocs-ci/issues/356

More information is available at old PR below.
Pulled from PR: https://github.com/red-hat-storage/ocs-ci/pull/185

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -98,6 +110,73 @@ def ceph_pool_factory_class(request):
...
+    cluster_version, image_dict = get_ocs_version()
+    file_name = os.path.join(
+        config.ENV_DATA['cluster_path'],
+        "ocs_version." + datetime.now().isoformat())
+    with open(file_name, "w") as file_obj:
+        report_ocs_version(cluster_version, image_dict, file_obj)
+    log.info("human readable ocs version info written into %s", file_name)
+
+
+@pytest.fixture(scope="session", autouse=True)
```
Comment (ebenahar) [inline tests/conftest.py]: This fixture already exists in this file - https://github.com/red-hat-storage/ocs-ci/blob/9c9a814c7a847a4bee9b84112640c271dd2e9118/tests/conftest.py#L703

Comment (vasukulkarni) [reply tests/conftest.py]: Fixed

**Comment by vasukulkarni** [comment]
Going to ammend commits to add sign.

File: ocs_ci/ocs/version.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        image_dict.setdefault(ns, ns_dict)
+
+    logger.debug("ocs version collected")
+
+    return cluster_version, image_dict
+
+
+def report_ocs_version(cluster_version, image_dict, file_obj):
+    """
+    Report OCS version via:
```
Comment (clacroix12) [inline ocs_ci/ocs/version.py]: This should be a google style docstring that includes the args.

Comment (vasukulkarni) [reply ocs_ci/ocs/version.py]: @clacroix12 I have fixed this in my last commit, hopefully its correct, Please review. Thanks

File: ocs_ci/ocs/version.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+     * python logging
+     * printing human readable version into file_obj (stdout by default)
+
+    Args:
+    cluster_version (dict): cluster version dict
+    image_dict (dict): dict of image objects
+    file_obj (object):  file object to log information
```
Comment (clacroix12) [inline ocs_ci/ocs/version.py]: These args should be indented:
```python
    Args:
       cluster_version (dict): cluster version dict
       image_dict (dict): dict of image objects
       file_obj (object):  file object to log information
```

Comment (vasukulkarni) [reply ocs_ci/ocs/version.py]: Fixed

**Pull request merged**